sr_external_dependencies/include/sr_external_dependencies/external/common/tactile_edc_ethercat_protocol.h

```diff
@@ -136,20 +144,20 @@ typedef enum                                                            // Data
 
     FROM_TACTILE_SENSOR_TYPE_BIOTAC_NUM_VALUES = 0x001C
 }FROM_TACTILE_SENSOR_TYPE_BIOTAC;
-
-
+
+
 typedef enum                                                   // Data you can request from MCP320x ADC
 {
     TACTILE_SENSOR_TYPE_MCP320x_INVALID        = 0x0000,       //!<
     TACTILE_SENSOR_TYPE_MCP320x_TACTILE        = 0x0001       //!< Only one type of sensor
-} FROM_TACTILE_SENSOR_TYPE_MCP320x;
+} FROM_TACTILE_SENSOR_TYPE_MCP320x;
 
 
 typedef struct
 {
     int16u  Pac[2];
     int16u  other_sensor_0;
-    //int16u  other_sensor_1;                               #ERROR NO! Do not place this here
+    //int16u  other_sensor_1;                               #ERROR NO! Do not place this here
```

Comment:
This line is shown as changed either because of auto removed trailing whitespace by IDE or but fixing Windows -> Linux new line ending.
I think it makes sense to keep this commented line as it shows what should not be done when adding a new sensor. Most likely because of overall limited size of EtherCAT packet.

Reply:
ok :+1: